### PR TITLE
Update: added zip next steps

### PR DESCRIPTION
**Pull request by rhoads-zach**
@edewit let me know what you think

### CODE REVIEW

File: docs/topics/launcher-review-zip.adoc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.Next Steps: Run your Booster.
+
+Extract your booster:
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ uznip {ZIP_CREATE}.zip
```
Comment (gastaldi) [inline docs/topics/launcher-review-zip.adoc]: Typo here. Should be `unzip`

**Comment by rhoads-zach** [comment]
resolves #419

**Comment by gastaldi** [comment]
LGTM

**Comment by rhoads-zach** [comment]
merged: b0591fe051d4c01fc801b57e1a88ff7fdcff86a8

**Pull request closed without merging**